tests/schema_test.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for the Wobbly database schema."""
+
+from __future__ import annotations
+
+import subprocess
+
+import pytest
+from safir.database import create_database_engine, unstamp_database
+
+from wobbly.config import config
+from wobbly.schema import SchemaBase
+
+
+@pytest.mark.asyncio
+async def test_schema() -> None:
+    """Test for any unmanaged schema changes.
+
+    Compare the current database schema in its SQLAlchemy ORM form against a
+    dump of the SQL generated from the last known Alembic migration and ensure
+    that Alembic doesn't detect any schema changes.
+    """
+    engine = create_database_engine(
+        config.database_url, config.database_password
+    )
+    async with engine.begin() as conn:
+        await conn.run_sync(SchemaBase.metadata.drop_all)
+    await unstamp_database(engine)
+    await engine.dispose()
+    subprocess.run(["alembic", "upgrade", "head"], check=True)
+    subprocess.run(["alembic", "check"], check=True)
```

Comment:
I'm not sure I see the benefit, but maybe I'm missing something? It feels like it's more code and potentially more fragile because the internal API might change.

Reply:
Sometimes there could be weirdness with finding `alembic` depending on how paths and venvs and such are configured in folks' local envs, but I have no evidence that will be the case here, and if it works with the blessed way of setting that stuff up, then it really doesn't matter. Agreed also that it's more fragile to Alembic changes.

I rescind the suggestion!

Reply:
Ah, yes, that makes sense! I think we're saved here because we tell people to always run tests with `tox`, which should make sure everything is set up properly and blocks running commands that aren't part of the venv.